llvm/test/CodeGen/X86/asm-mismatched-types.ll

```diff
@@ -8,252 +8,252 @@ target triple = "x86_64--"
 ; void foo(int p) { register int reg __asm__("r8") = p;
 ; __asm__ __volatile__("# REG: %0" : : "r" (reg)); }
 
-; CHECK-LABEL: reg64_as_32:
+; CHECK-LABEL: {{^}}reg64_as_32:
```

Comment:
Why this change?

Reply:
> Why this change?

Yes, this is not a necessary modification.